src/components/Navbar/Navbar.jsx

```diff
@@ -55,24 +60,28 @@ const Navbar = ({ menuRequired, navTxt, logoRemove ,prevPage}) => {
                 <BiSearch className="search-btn" />
               </div>
             </div>
-            {navIcons.map((navIcon) => {
+
+            {navIconsData.map((navIcon) => {
```

Comment:
suggestion - you can avoid parenthesis if it's just one argument. For code readability.